file.c

```diff
@@ -4094,6 +4094,25 @@ rb_file_s_absolute_path(int argc, const VALUE *argv)
     return rb_file_absolute_path(argv[0], argc > 1 ? argv[1] : Qnil);
 }
 
+/*
+ *  call-seq:
+ *     File.absolute_path?(file_name)  ->  true or false
+ *
+ *  Returns <code>true</code> if +file_name+ is an absolute path, and
+ *  <code>false</code> otherwise.
+ *
+ *     File.absolute_path?("c:/foo")     #=> false (on Linux), true (on Windows)
+ */
+
+VALUE
+rb_file_s_absolute_path_p(VALUE klass, VALUE fname)
```

Comment:
Should be `static`?

Reply:
Made this one `static` and removed `rb_file` prefixes :+1:.